Update GainingPrivacyWasabi.md (#408)
zkSNACKs/WasabiDoc: docs/why-wasabi/GainingPrivacyWasabi.md

## Technology behind Wasabi

- It has [advanced coin control features](/FAQ/FAQ-UseWasabi.md#coin-control-best-practices) to avoid linking coins from different transactions.

> - It has [advanced labeling features](/using-wasabi/Receive.md#the-importance-of-labeling) to tell who is/are the one/s that know about specific coins.

Reviewer: ```suggestion
- It has [advanced labeling features](/using-wasabi/Receive.md#the-importance-of-labeling) to tell which entities know about specific coins.
```
